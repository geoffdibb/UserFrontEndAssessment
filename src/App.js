import React, { Component } from 'react';
import './App.css';


import Navbar from "./navbar";
export class App extends Component {

  render() {
    return (
      <div  style={{ display: 'flex', justifyContent: 'center', backgroundColor: "black", color: "white" }}>
        <Navbar/>

        
      </div>
    );
  }


}

export default App;