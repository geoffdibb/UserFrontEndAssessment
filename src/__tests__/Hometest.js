import React from 'react';
import Home from '../Home';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import App from '../App';


it('snapshot Home', () => {
  const tree = renderer.create(<Home/>).toJSON()
  expect(tree).toMatchSnapshot();
})

it('renders Home text', () => {
  const wrapper = shallow(<App />);
      const Home = <Home/>;
  expect(wrapper.contains(Home)).toEqual(true);

});