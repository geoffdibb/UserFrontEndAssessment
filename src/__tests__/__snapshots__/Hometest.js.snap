// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot Home 1`] = `
<div>
  <h2>
    Welcome Home 
  </h2>
     
</div>
`;
